packages/web/src/common/store/lineup/sagas.js

```diff
@@ -46,6 +49,10 @@ function* filterDeletes(tracksMetadata, removeDeleted) {
   const remoteConfig = yield getContext('remoteConfigInstance')
   yield call(remoteConfig.waitForRemoteConfig)
 
+  const isUSDCGatedContentEnabled = yield getFeatureEnabled(
```

Comment:
Note: In the PR I based this on, there was a `waitForRead()` at this point. Looking into what `waitForRead` is waiting on (reachability and account), I don't think there's any additional need to wait on that condition here if the original lineup processing wasn't already waiting for it.
But if others can think of a reason why we should, I'm happy to add it.